CONTRIBUTING.md

```diff
@@ -82,16 +82,28 @@ The test module, clusterctl, and experiments do not provide any backward compati
 
 #### Backporting a patch
 
-We only accept backports of critical bugs, security issues, or bugs without easy workarounds, any
-backport MUST not be breaking for either API or behavioral changes. In order to improve user/developer experience
-maintainers can choose to backport:
-- Doc improvements
+We generally do not accept PRs directly against release branches, while we might accept backports of fixes/changes already
+merged into the main branch.
+
+Any backport MUST not be breaking for either API or behavioral changes. 
+
+We generally allow backport of following changes to all the supported branches:
+- Critical bugs fix, security issues fix, or fix for bugs without easy workarounds.
+- Dependency bumps for CVE (usually limited to CVE resolution; backports of non-CVE related version bumps are considered exceptions to be evaluated case by case)
+- Cert-manager version bump (to avoid having branch using cert-manager versions out of support, when possible)
+- Changes required to support new Kubernetes versions, when possible. See [supported Kubernetes versions](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions) for more details.
+
+We generally allow backport of following changes only to the latest supported branch:
+- Improvements to existing docs (the latest stable version hosts the current version of the book)
 - Improvements to CI signal
-- Improvements to the test framework (enabling improvements to provider's test signal)
-- Cert-manager bump (to avoid having branch using cert-manager versions out of support, when possible)
-- Changes required to support new Kubernetes versions, when possible.
+- Improvements to the test framework
+
+Like any other activity in the project, backporting a fix/change also requires that someone volunteers for the task.
+The cherry pick bot can be used for automating some of this work thus reducing the effort for this job. But
+independent of that, the community is expected to also help in taking care of older Cluster API release branches while still
```

Comment:
> the community is expected to also help in taking care of older Cluster API release branches 

not sure this sentence is necessary, it's a little vague

What do you think of this instead?

> Like any other activity in the project, backporting a fix/change is a community-driven effort and requires that someone volunteers to own the task. In most cases, the [cherry-pick bot](https://pkg.go.dev/github.com/k8s-ci-robot/test-infra/prow/external-plugins/cherrypicker) can (and should) be used to automate opening a PR.